src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -33,16 +34,6 @@
 
 public class Scheduler implements org.apache.mesos.Scheduler, Runnable {
     
-  public static final String NAMENODE_INIT_MESSAGE = "-i";
-  public static final String NAMENODE_BOOTSTRAP_MESSAGE = "-b";
-  private static final String NAME_NODE_ID = "namenode";
-  private static final String JOURNAL_NODE_ID = "journalnode";
-  private static final String DATA_NODE_ID = "datanode";
-  private static final String ZKFC_NODE_ID = "zkfc";
-  private static final String NODE_EXECUTOR_ID = "NodeExecutor";
-  private static final String NAME_NODE_EXECUTOR_ID = "NameNodeExecutor";
-  private static final String NAME_NODE_TASK_STR = NAME_NODE_ID + "." + NAME_NODE_ID + "."
-      + NAME_NODE_EXECUTOR_ID;
   private static final Integer TOTAL_NAME_NODES = 2;
```

Comment:
Why does this constant stay here? It should probably be configurable at some point, but might as well go into HDFSConstants with the rest